webcomponents/tool/pom.xml

```diff
@@ -130,6 +130,15 @@
                             <npmVersion>${sakai.npm.version}</npmVersion>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>npm cache clean</id>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+                        <configuration>
+                            <arguments>cache clean --force</arguments>
+                        </configuration>
```

Comment:
@hornersa I would tie this to the clean phase so that when perform `mvn clean` it will run.